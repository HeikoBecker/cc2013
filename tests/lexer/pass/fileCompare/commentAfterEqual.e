commentAfterEqual.c:1:1: keyword int
commentAfterEqual.c:1:5: identifier main
commentAfterEqual.c:1:9: punctuator (
commentAfterEqual.c:1:10: keyword int
commentAfterEqual.c:1:14: identifier argc
commentAfterEqual.c:1:18: punctuator ,
commentAfterEqual.c:1:20: keyword char
commentAfterEqual.c:1:24: punctuator *
commentAfterEqual.c:1:26: identifier argv
commentAfterEqual.c:1:30: punctuator [
commentAfterEqual.c:1:31: punctuator ]
commentAfterEqual.c:1:32: punctuator )
commentAfterEqual.c:1:34: punctuator {
commentAfterEqual.c:2:2: keyword char
commentAfterEqual.c:2:6: punctuator *
commentAfterEqual.c:2:8: identifier text
commentAfterEqual.c:2:13: punctuator =
commentAfterEqual.c:2:15: stringliteral "hihoha"
commentAfterEqual.c:2:23: punctuator ;
commentAfterEqual.c:3:2: keyword char
commentAfterEqual.c:3:7: identifier t
commentAfterEqual.c:3:9: punctuator =
commentAfterEqual.c:3:9: punctuator =
commentAfterEqual.c:3:39: constant 'o'
commentAfterEqual.c:3:42: punctuator ;
commentAfterEqual.c:4:2: keyword return
commentAfterEqual.c:4:9: constant 0
commentAfterEqual.c:4:10: punctuator ;
commentAfterEqual.c:5:1: punctuator }
