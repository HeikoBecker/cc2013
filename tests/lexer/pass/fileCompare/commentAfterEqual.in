int main(int argc, char* argv[]) {
	char* text = "hihoha";
	char t =/*error: comment int union */'o';
	return 0;
}
