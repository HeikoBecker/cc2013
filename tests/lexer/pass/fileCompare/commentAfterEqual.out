/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:1: keyword int
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:5: identifier main
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:9: punctuator (
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:10: keyword int
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:14: identifier argc
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:18: punctuator ,
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:20: keyword char
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:24: punctuator *
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:26: identifier argv
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:30: punctuator [
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:31: punctuator ]
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:32: punctuator )
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:1:34: punctuator {
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:2:2: keyword char
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:2:6: punctuator *
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:2:8: identifier text
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:2:13: punctuator =
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:2:15: stringliteral "hihoha"
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:2:23: punctuator ;
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:3:2: keyword char
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:3:7: identifier t
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:3:9: punctuator =
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:3:9: punctuator =
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:3:39: constant 'o'
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:3:42: punctuator ;
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:4:2: keyword return
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:4:9: constant 0
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:4:10: punctuator ;
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/commentAfterEqual.in:5:1: punctuator }
